wdl2cwl/main.py

```diff
@@ -500,6 +499,12 @@ def get_input(
 
             inputs.append(cwl.CommandInputParameter(id=input_name, type=input_type))
 
+        elif i[0] in record_schemas.keys():
+            inputs.append(cwl.CommandInputParameter(id=input_name, type=record_schemas.get(i[0])))
+
+        else:
+            raise ValueError(f"Failed to get input type: {i[0]}")
```

Comment:
Now raising an error if we know we cannot handle the input type (also the output in another function).

Reply:
Some unit tests failed, so I reverted this change. Added the `elif` that checks if it is a struct before the `else` block, this last one is handling the types in the same way as before. Only addition will be the extra `elif key in structs...`